test/IECore/IndexedIO.py

```diff
@@ -662,12 +662,13 @@ def testDefaultCompressionIsOff( self ):
 
 		size = os.path.getsize( filePath )
 
-		self.assertTrue( (size > (1024 * 4)) and (size < (1024 * 4 + 128)) )
+		self.assertTrue( ( size > (1024 * 4 ) ) )
+		self.assertTrue( ( size < (1024 * 4 + 512 ) ) )
```

Comment:
I suspect this is because the index is small it either doesn't compress or the compression ratio isn't as good with such small input data compared to gzip. 

In all practical cases were slightly smaller than before.

Reply:
Okie dokie